solr/core/src/java/org/apache/solr/packagemanager/DefaultPackageRepository.java

```diff
@@ -101,16 +106,21 @@ public Path download(String artifactName) throws SolrException, IOException {
   }
 
   private void initPackages() {
-    try (CloseableHttpClient client = HttpClientBuilder.create().build()) {
+    try (Http2SolrClient client =
+        new Http2SolrClient.Builder(repositoryURL).useHttp1_1(true).build()) {
```

Comment:
When not specifying the http1.1 as protocol I got an `java.io.IOException: frame_size_error/invalid_frame_length` exception which I figured is because the response's format does not support http2. Using http1 works well. 

Do you think making a change in the response so that we can use http2 here would be a neater solution?

Reply:
I wonder how the HTTP protocol change (a low level thing) alters the the higher level response format?  Seems like there is something to root-cause there.

Reply:
Sorry it seems like I forgot to answer this. The reason why we need http 1.1 here is because actually we are not talking to Solr here, but the repository server. I am not really sure at this point if we should use a Solr client for this communication at all, seems quite silly. 
Although I have tried an implementation where I was using a plain jetty client here, instead of Solr but it came out rather ugly and a bit complicated with all the random parameters I had to set a default for.

On the other hand I tried changing the test (PackageManagerCLITest) to create an http 2 compatible web server for the repository server [here](https://github.com/apache/solr/blob/11253f05cfb31f9fb945c831d8889b3db1e607f1/solr/core/src/test/org/apache/solr/cloud/PackageManagerCLITest.java#L239-L256) but then I realised that probably repository servers in reality are not prepared to be compatible with http2 so just changing the test did not seem a viable option.

Reply:
@dsmiley  does this respnose make sense to you?   Do we need to change things up more?   Or is adding a comment sufficient?

Reply:
@bszabo97 there are other places where we create a straigh up httpclient and do a get..  Maybe we should just do that here?

Reply:
(pending)

Reply:
You got it working; seems fine.  A comment would be helpful to future readers.